great_expectations/data_context/data_context.py

```diff
@@ -893,6 +919,30 @@ def create_expectation_suite(self, expectation_suite_name, overwrite_existing=Fa
 
         return expectation_suite
 
+    def delete_expectation_suite(self, expectation_suite_name):
+        """Delete specified expectation suite from data_context expectation store.
+
+        Args:
+            expectation_suite_name: The name of the expectation_suite to create
+
+        Returns:
+            True for Success and False for Failure.
+        """
+        expectation_suite = ExpectationSuite(expectation_suite_name=expectation_suite_name)
+        key = ExpectationSuiteIdentifier(expectation_suite_name=expectation_suite_name)
+        del self._stores[self.expectations_store_name][key]
```

Comment:
use the store remove_key method

Reply:
This had the remove_key in previous iteration which remains commented. Uncommenting and moving back to that version.